build.sbt

```diff
@@ -64,8 +64,13 @@ lazy val all = project
   .enablePlugins(NoPublishPlugin) // don't publish
 
 // a published artifact aggregating API docs for viewing at javadoc.io
+// build and view scaladocs locally:
+// sbt unidocs/doc
+// view at:  file:///your/path/to/coulomb/unidocs/target/scala-3.1.2/unidoc/index.html
```

Comment:
Annoyingly this path will change with scala version updates :)

I will look into adding a helper method to sbt-typelevel for this.
- https://github.com/typelevel/sbt-typelevel/issues/270